lib/Sema/CSApply.cpp

```diff
@@ -1233,11 +1233,32 @@ namespace {
                          ConstraintLocatorBuilder locator,
                          bool isImplicit, AccessSemantics semantics) {
       // Determine the declaration selected for this subscript operation.
-      auto selected = getOverloadChoice(
+      auto selected = getOverloadChoiceIfAvailable(
```

Comment:
Huh, this makes me wonder: are *all* direct uses of `getOverloadChoice` within `CSApply.cpp` suspect because of this issue?